### ISSUE TITLE
Pipedrive Integration - Phone field issue

### ISSUE THREAD

**Issue by ghost** [verbb/formie#823]
**Description**
We have a Craft CMS headless setup with NextJS, and recently upgraded from 1.4.18 to 1.5.12. We did our usual testing with the forms, and then discovered that after the update, every new pipedrive entry had two commas appended to the phone number field.

This issue is happening somewhere in the process of submitting to pipedrive, because the submission on the Craft CMS backend/database doesn't have it.

We have made a workaround that fixes it temporarily. See below.
```php
private function _removeCommasFromPhone()
{
    Event::on(Phone::class, Phone::EVENT_MODIFY_VALUE_FOR_INTEGRATION,
        function (ModifyFieldIntegrationValueEvent $event) {
            $event->value = $event->value['number'];
        });
}
```

We also tried to update to the latest versions that were released around 12 hours ago, but that didn't help.

**Steps to reproduce**
1. Upgrade to newest formie version
2. Run integration with Pipedrive

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.5.12
- Craft version: 3.7.37
- Multi-site: No

**Additional context**

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.13.2",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.14](https://github.com/verbb/formie/releases/tag/1.5.14)
